scripts/apothecary/formulas/cairo.sh

```diff
@@ -147,13 +158,13 @@ function copy() {
 			mkdir -p $1/lib/$TYPE/Win32
 			cp -v Cairo-VS/projects/Release/cairo.lib $1/lib/$TYPE/Win32/cairo-static.lib
 			cp -v Cairo-VS/projects/Release/pixman.lib $1/lib/$TYPE/Win32/pixman-1.lib
-			cp -v Cairo-VS/projects/Release/libpng.lib $1/lib/$TYPE/Win32/libpng.lib
+			cp -v Cairo-VS/libs/libpng.lib $1/lib/$TYPE/Win32
 		elif [ $ARCH == 64 ] ; then
 			# make the libs path 
 			mkdir -p $1/lib/$TYPE/x64
 			cp -v Cairo-VS/projects/x64/Release/cairo.lib $1/lib/$TYPE/x64/cairo-static.lib
 			cp -v Cairo-VS/projects/x64/Release/pixman.lib $1/lib/$TYPE/x64/pixman-1.lib
-			cp -v Cairo-VS/projects/x64/Release/libpng.lib $1/lib/$TYPE/Win32/libpng.lib
+			cp -v Cairo-VS/libs/libpng.lib $1/lib/$TYPE/x64
 		fi
```

Comment:
I changed VS-Cairo to compile libpng and zlib because the project included with libpng only compiled 32bit so the folder structure changed a little as the build step moves the libraries into the lib folder for cairo and pixman to use. I could probably have it copy them instead of move them but currently they aren't in the release folder after compiling